arcade/draw_commands.py

```diff
@@ -706,10 +743,17 @@ def draw_rectangle_outline(center_x: float, center_y: float, width: float,
     _generic_draw_line_strip(point_list, color, gl.GL_TRIANGLE_STRIP)
 
 
+@warning(
+    warning_type=NameChangeWarning,
+    new_name="draw_lrbt_rectangle_filled"
+)
 def draw_lrtb_rectangle_filled(left: float, right: float, top: float,
                                bottom: float, color: Color):
     """
-    Draw a rectangle by specifying left, right, top, and bottom edges.
+    Draw a rectangle by specifying left, right, top and bottom edges.
+
+    .. deprecated:: 3.0
+        Use :py:func:`draw_lrbt_rectangle_filled` instead!
```

Comment:
Same as the previous comment.
```suggestion
       Use :py:func:`draw_lrbt_rectangle_filled` instead!
```